.github/workflows/test.yml

```diff
@@ -73,5 +73,5 @@ jobs:
         env:
           BONSAI_API_URL: ${{ secrets.BONSAI_API_URL }}
           BONSAI_API_KEY: ${{ secrets.BONSAI_API_KEY }}
-          BONSAI_PROVING: bonsai
+          RISC0_DEV_MODE: false
```

Comment:
I'm not sure about this, but I don't think the zkvm handles `false` currently. It only knows about whether the `RISC0_DEV_MODE` env var is set or not (it doesn't care what value it's set to). See: https://github.com/risc0/risc0/pull/752#discussion_r1292350619

Reply:
Yes you are correct, there are still inconsistencies between the usage of `dev_mode` here and in the zkvm. However, this codebase uses it this way: 
https://github.com/risc0/bonsai-foundry-template/blob/8838e05c45025ced74c8bad0f6d1d88fc13f82a6/relay/src/lib.rs#L187-L193